libs/shared/feature/form-nuv/src/lib/components/forms/builder/form-builder.component.ts

```diff
@@ -154,7 +156,8 @@ export class FormBuilderComponent implements DoCheck, OnDestroy {
 
   public intakeRendererOptions: NuvalenceFormRendererOptions = AdminBuilderIntakeRendererOptions;
   public reviewRendererOptions: NuvalenceFormRendererOptions = AdminBuilderReviewRendererOptions;
-  public formData$?: Observable<Record<string, unknown>> = of({});
+  private readonly formData: BehaviorSubject<Record<string, unknown>> = new BehaviorSubject({});
```

Comment:
Our linting should catch this but isn't right now, but private should be underscored
```suggestion
  private readonly _formData: BehaviorSubject<Record<string, unknown>> = new BehaviorSubject({});
```